docs/getting-started.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 2
+sidebar_label: Getting Started
+title: "Getting Started"
+keywords:
+- Harvester
+- getting started
+- step by step
+- deployment
+---
+
+A three-node [Harvester cluster](../glossary/#harvester-cluster) is required to fully realize multi-node features such as high availability. Certain versions of Harvester allow you to create clusters with two management nodes and one [witness node](../advanced/witness/) (and optionally, one or more worker nodes). You can also create [single-node clusters](../advanced/singlenodeclusters/) that support most Harvester features (excluding high availability, multi-replica support, and live migration). 
+
+This guide walks you through the steps required to deploy a high-availability cluster and virtual machines (VMs) that can host [guest clusters](../glossary/#guest-cluster--guest-kubernetes-cluster). 
+
+## 1. Verify that the minimum hardware and network requirements for installing and running Harvester are met. 
```

Comment:
The existing wording emphasizes the need to meet the requirements (instead of just checking the list). If you think the sentence is too long, we can shorten it to `Verify that the minimum hardware and network requirements are met.`.

Reply:
Cool, thanks!